contracts/APIConsumer.sol

```diff
@@ -69,6 +69,17 @@ contract APIConsumer is ChainlinkClient {
         volume = _volume;
     }
 
+    /**
+     * Withdraw LINK from this contract
+     * 
+     * NOTE: DO NOT USE THIS IN PRODUCTION AS IT CAN BE CALLED BY ANY ADDRESS.
+     * THIS IS PURELY FOR EXAMPLE PURPOSES ONLY.
```

Comment:
I think this is bad enough practice that we should just make the contract implement Ownable and put an `onlyOwner` modifier on functions.